store/router.go

```diff
@@ -195,6 +209,14 @@ func (r *Router) multiSourceRead(ctx context.Context, commitment []byte, fallbac
 	}
 
 	key := crypto.Keccak256(commitment)
+
+	// check if key is an RLP encoded certificate, if not, assume it's an s3 key
+	var cert verify.Certificate
+	err := rlp.DecodeBytes(commitment, &cert)
+	if err != nil {
+		key = commitment
+	}
+
```

Comment:
Need to change verify logic on line 233. When `key == commitment` (ie. EigneDA failes), verify that `keccak256(data) == key`

https://github.com/polymerdao/eigenda-proxy/pull/1/files#diff-fc787786f5f17942139efa2d18ffd7b028c888c628e07ecc4ce737d917f09eefR232-R237